class.photon.php

```diff
@@ -534,7 +541,9 @@ public static function filter_the_content( $content ) {
 						}
 
 						// Tag an image for dimension checking
-						$new_tag = preg_replace( '#(\s?/)?>(\s*</a>)?$#i', ' data-recalc-dims="1"\1>\2', $new_tag );
+						if ( ! Jetpack_AMP_Support::is_amp_request() ) {
```

Comment:
Could you add a `class_exists` check here before to use `Jetpack_AMP_Support`? That class does not exist on WordPress.com yet, unfortunately, so we'd need to check for it before we use it in files that are also in use on WordPress.com.